.nvmrc

```diff
@@ -0,0 +1 @@
+lts/*
```

Comment:
if you're using `nvm`, `nvm install` will give you the latest long term support node version, and then you can `nvm use` to get that version in your shell. Cloudflare listens to this file too https://developers.cloudflare.com/pages/configuration/build-image/